scripts/docker-entrypoint.sh

```diff
@@ -277,9 +279,12 @@ unset DB_USERNAME DB_PASSWORD SECRET_DB_USERNAME SECRET_DB_PASSWORD CASSANDRA_US
 
 unset pega_root lib_root config_root
 
+
 # Run tomcat if the first argument is run otherwise try to run whatever the argument is a command
-if [ "$1" = 'run' ]; then
-  exec catalina.sh "$@"
-else
-  exec "$@"
-fi
+if [ ! -z "$1" ]; then
+  if [ "$1" = 'run' ]; then
+    exec gosu tomcat catalina.sh "$@"
+  else
+    exec gosu tomcat "$@"
```

Comment:
Maybe keep this exec "$@" in case there is a use case to continue to be able to run catalina.sh with current user?  One example might be mounting in a git repo for local development purposes, or there might be other requirements for Pega to be able to read/write to the filesystem.